elliottlib/cli/attach_cve_flaws_cli.py

```diff
@@ -100,12 +112,32 @@ def attach_cve_flaws_cli(runtime, advisory_id, noop, default_advisory_type):
     runtime.logger.info(f'Bugs already attached: {len(existing_bug_ids)}')
     runtime.logger.info(f'New bugs ({len(new_bugs)}) : {sorted(new_bugs)}')
 
-    if new_bugs:
-        advisory.addBugs(flaw_ids)
-    if noop:
-        return True
-
-    return advisory.commit()
+    if not noop:
+        if new_bugs:
+            runtime.logger.info('Attaching bugs %s', flaw_ids)
+            advisory.addBugs(flaw_ids)
+        runtime.logger.info("Updating advisory %s", advisory_id)
+        advisory.commit()
+
+    runtime.logger.info('Associating CVEs with builds')
+    errata_api = AsyncErrataAPI(errata_config.get("server", constants.errata_url))
+    try:
+        await errata_api.login()
+        await associate_builds_with_cves(errata_api, advisory, attached_tracker_bugs, tracker_flaws, flaw_id_bugs, noop)
+    finally:
+        await errata_api.close()
+
+
+async def associate_builds_with_cves(errata_api: AsyncErrataAPI, advisory: Erratum, attached_tracker_bugs: List[Bug], tracker_flaws: Dict[int, List[int]], flaw_id_bugs: Dict[int, Bug], dry_run: bool):
+    attached_builds = [b for pv in advisory.errata_builds.values() for b in pv]
+    rpm_cves: Dict[Bug, str] = bzutil.get_valid_rpm_cves(attached_tracker_bugs)
```

Comment:
Why does this exclude container cves? Does this feature work with CVE for containers?